packages/admin-panel/src/index.js

```diff
@@ -1,33 +1,34 @@
-/**
- * Tupaia MediTrak
- * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
-
-import React from 'react';
+import React, { lazy, Suspense } from 'react';
+import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
 import { render as renderReactApp } from 'react-dom';
-import { MuiThemeProvider, StylesProvider } from '@material-ui/core/styles';
-import { ThemeProvider } from 'styled-components';
-import CssBaseline from '@material-ui/core/CssBaseline';
-import { Provider } from 'react-redux';
-import { PersistGate } from 'redux-persist/lib/integration/react';
 import 'react-table/react-table.css';
+import AdminPanel from './App';
+import { AdminPanelProviders, VizBuilderProviders } from './utilities';
+import { Footer, Navbar } from './widgets';
 
-import { theme } from './theme';
-import { store, persistor } from './store';
-import { App } from './App';
+const VizBuilder = lazy(() => import('./vizBuilder'));
 
 renderReactApp(
-  <Provider store={store}>
-    <PersistGate persistor={persistor} loading={null}>
-      <StylesProvider injectFirst>
-        <MuiThemeProvider theme={theme}>
-          <ThemeProvider theme={theme}>
-            <CssBaseline />
-            <App />
-          </ThemeProvider>
-        </MuiThemeProvider>
-      </StylesProvider>
-    </PersistGate>
-  </Provider>,
+  <Router>
+    <Suspense fallback={<div>loading...</div>}>
+      <Switch>
+        <Route path="/viz-builder" exact>
+          <VizBuilderProviders>
```

Comment:
Viz Builder has its own providers so that there is no cross pollination between it and admin panel so that:
- it can be exported as a standalone app
- hot-reloading works
- can have a different theme if need be
- can easily add other providers like react-query provider etc
- better code splitting